### PR TITLE
Rework design table and L1 column structure

### PR DESCRIPTION
**Pull request by bpbond**
Our initial L1 data (v0-9) had a lot of information shoehorned into the "design_link" column. This column was a pain for users to parse; difficult to work with or update for us; and made it hard to spot mistakes.

This PR changes the design table structure so that individual columns encode information about
* `Instrument` - the overall measurement instrument (e.g., "AquaTROLL")
* `Which` - which specific AquaTROLL, etc.,
* `Individual` - either which experiment unit (e.g. tree) is being measured, or which named TEROS sensor, etc.
* `research_name` - as before, this describes the `value` column, i.e. what is being measured

In summary, this makes working with the design table easier for us, and working with the L1 data easier for everyone.

## Old L1 data structure

Screenshot 2024-04-02 at 9 38 51 AM

## New structure (this PR)

Screenshot 2024-04-02 at 9 40 23 AM

### CODE REVIEW

**Comment by bpbond** [comment]
When you get to this SP-

This is our last chance to change the L1 structure without breaking things--that is, when we declare it "v1-0", people will assume and expect that it's stable for a while.

With that in mind, we might want to consider changing some of the column names above:

* `research_name` - what about "Variable"?
* `Instrument` - any better ideas?
* `Which` - any better ideas?
* `Individual` - ?
* `value` - to "Value" (for consistent capitalization)?

Thx

**Comment by stephpenn1** [comment]
Okay - I'm running this on my computer so I can inspect each part and will give full feedback after

**Comment by bpbond** [comment]
@stephpenn1 Is it OK if we merge this and then deal with reworking columns ( #127 ) via a second PR? If you approve, that would be helpful, as I have some other changes that depend on this. Thanks

**Review by stephpenn1 — APPROVED** [review]
We will revisit the new columns, approving this for now for merging

**Comment by bpbond** [comment]
Thank you!
